apps/etterlatte-migrering/src/main/kotlin/migrering/MigrerSpesifikkSak.kt

```diff
@@ -51,7 +51,7 @@ internal class MigrerSpesifikkSak(
         context: MessageContext,
     ) {
         val sakId = packet.sakId
-        if (pesysRepository.hentStatus(sakId) != null) {
+        if (pesysRepository.hentStatus(sakId) == Migreringsstatus.FERDIG) {
```

Comment:
Hva skjer egentlig her hvis vi har forsøkt en migrering tidligere som har feilet? 

Tanken er vel at det kanskje skal ligge en "manuell" oppgave om å migrere ferdig her? Eller burde vi egentlig avbryte den behandlingen vi har forsøkt å migrere, men som feilet (i alle fall frem til all logikken er ferdig)?

Hvis behandlingen ikke avbrytes, får man da lov å forsøke å migrere på nytt når det allerede finnes en førstegangsbehandling på saken?

Reply:
Følgjer opp det meir i EY-2819, liker ideen (som vi snakka om i går) med å avbryte behandlinga viss migreringa feilar, og å opne for å da migrere på nytt.